clients/banking/src/components/MerchantProfilePaymentList.tsx

```diff
@@ -0,0 +1,271 @@
+import { ActionCell, Cell, CopyableTextCell, HeaderCell } from "@swan-io/lake/src/components/Cells";
+import { Icon } from "@swan-io/lake/src/components/Icon";
+import { LakeText } from "@swan-io/lake/src/components/LakeText";
+import {
+  ColumnConfig,
+  LinkConfig,
+  PlainListView,
+} from "@swan-io/lake/src/components/PlainListView";
+import { Tag } from "@swan-io/lake/src/components/Tag";
+import { colors, spacings } from "@swan-io/lake/src/constants/design";
+import { isNotNullish, isNullishOrEmpty } from "@swan-io/lake/src/utils/nullish";
+import dayjs from "dayjs";
+import { ReactElement, ReactNode } from "react";
+import { StyleSheet, View } from "react-native";
+import { match } from "ts-pattern";
+import { MerchantPaymentFragment } from "../graphql/partner";
+import { formatCurrency, t } from "../utils/i18n";
+
+const styles = StyleSheet.create({
+  cell: {
+    display: "flex",
+    paddingHorizontal: spacings[16],
+    flexGrow: 1,
+    flexDirection: "row",
+    alignItems: "center",
+    width: 1,
```

Comment:
Avoid cells with `width: 1`. Copy the logic I put in place recently.

Reply:
Actually there's no need for a `styles.cell` anymore, use directly `Cell` from `lake`